### DISCUSSION TITLE
How to call this service from Flutter (dart) code?

### DISCUSSION THREAD

**Discussion by EnvOut** [google/tarpc#447] [Q&A] [answered]
How to call this tarpc service from Flutter (dart) code?

**Comment by tikue** [comment, marked as answer]
Hey, sorry for not seeing this earlier!

tarpc doesn't generate stubs for any languages other than Rust. For cross-language RPCs, I'd probably stick with something like gRPC, where there is a compiler for multiple languages.

**Comment by djoyce-ts** [comment]
You can also write code that wraps the client and exports a C interface using Rust's C ABI support.
